### ISSUE TITLE
Missing getActionsAsObject from Actions

### ISSUE THREAD

**Issue by dariocravero** [acdlite/flummox#242]
Hi all,

There seems to be an issue with the new way of injecting actions into a FluxComponent, essentially, [the default getter being referenced, getActionsAsObject](https://github.com/acdlite/flummox/blob/master/src/addons/reactComponentMethods.js#L277) doesn't exist :).

Should it live somewhere in [createActions](https://github.com/acdlite/flummox/blob/master/src/createActions.js)?

I'm overcoming this now by defining a getter myself but it seems like a bit of an overkill :(

```
class ViewContainer extends FluxComponent {
  render() {
    return <FluxComponent
      actions={{router: ({navigate}) => ({navigate})}}
      stores={{router: ['keys', 'uri']}}><View /></FluxComponent>;
  }
}
```

Is it a bug?
Thanks,
Darío

**Comment by johanneslumpe** [comment]
@dariocravero this is a bug and has already been [fixed in master](https://github.com/acdlite/flummox/blob/master/src/addons/reactComponentMethods.js#L282) :)

**Comment by dariocravero** [comment]
@johanneslumpe sorry, I meant to refer to the line you pointed out and referred to the store's method instead. However, as I wrote above, that's only calling `getActionsAsObject`. Unless I'm missing something -perhaps it is in an unmerged branch?-, `getActionsAsObject` isn't defined anywhere else on the codebase and that would be the bug?

**Comment by johanneslumpe** [comment]
oh shoot! Man I shouldn't do stuff when I'm sleepy. You are right. It is fixed in one place, but still not in another. I will get this fixed asap :)

**Comment by johanneslumpe** [comment]
Ok, this is fixed with #248 - thanks for paying attention! :+1: Can you pull from master again and check? Sorry for an inconvenience caused by this!

**Comment by dariocravero** [comment]
That's perfect! :) Thanks! :dancer:
